py3-packaging.yaml

```diff
@@ -27,11 +35,36 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 85442b8032cb7bae72866dfd7782234a98dd2fb7
 
-  - runs: pip install . --prefix=/usr --root=${{targets.destdir}}
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-pyparsing
+      provider-priority: ${{range.value}}
+      provides:
+        - py3-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
 
-  - uses: strip
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
 
 update:
   enabled: true
+  ignore-regex-patterns:
+    - a
+    - b
+    - rc
+    - post
   github:
     identifier: pypa/packaging
```

Comment:
should we add a test pipeline for this too?

Reply:
once melange supports versioned python I'll fix the tests so it runs on all versions